Shell/main.cpp

```diff
@@ -561,6 +561,17 @@ int main(int argc, char** argv)
         endpwent();
     }
 
+    StringBuilder sb;
+    sb.append(g.home);
+    sb.append("/.history");
+    CFile history_file(sb.to_string());
+    rc = history_file.open(CIODevice::ReadWrite);
+    if (rc < 0)
```

Comment:
`CIODevice::open()` returns a `bool`, so the appropriate error value to check for is `false`.